### PR TITLE
CONCF-941 | As a developer I would like getCuratedContentSection resource to throw exceptions when there is error

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/CONCF-941

### CODE REVIEW

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 1]
```diff
@@ -407,9 +407,14 @@ public function getCuratedContentSection() {
...
 		if ( empty( $section ) ) {
 			$this->response->setVal( 'items', false );
-		} else {
-			$data = $this->getCuratedContentData( $section );
-			$this->response->setVal( 'items', $data[ 'items' ] );
+			throw new NotFoundApiException( 'Section is not set' );
+		}
+
+		$data = $this->getCuratedContentData( $section );
+		$this->response->setVal( 'items', $data[ 'items' ] );
```
Comment (rogatty) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: You should do this after checking `if ( empty( $data ) )`

**Comment by Warkot** [comment]
LGTM

**Comment by bognix** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Pull request merged**